tfjs-core/src/ops/to_pixels_test.ts

```diff
@@ -15,154 +15,22 @@
-    const x = tf.tensor3d([10, 20, 30, 40, 50, 60], [2, 1, 3], 'int32');
-
-    const data = await tf.browser.toPixels(x);
-    const expected = new Uint8ClampedArray([10, 20, 30, 255, 40, 50, 60, 255]);
-    expect(data).toEqual(expected);
-  });
-
-  it('draws a rank-3 float32 tensor, 4 channel', async () => {
-    const x = tf.tensor3d(
-        [.05, .1001, .15, .2, .25, .3001, .35, .4], [2, 1, 4], 'float32');
-
-    const data = await tf.browser.toPixels(x);
-    const expected = new Uint8ClampedArray([
-      Math.round(.05 * 255), Math.round(.1001 * 255), Math.round(.15 * 255),
-      Math.round(.20 * 255), Math.round(.25 * 255), Math.round(.3001 * 255),
-      Math.round(.35 * 255), Math.round(.4 * 255)
-    ]);
-    expect(data).toEqual(expected);
-  });
-
-  it('draws a rank-3 int32 tensor, 4 channel', async () => {
-    const x = tf.tensor3d([10, 20, 30, 40, 50, 60, 70, 80], [2, 1, 4], 'int32');
-
-    const data = await tf.browser.toPixels(x);
-    const expected = new Uint8ClampedArray([10, 20, 30, 40, 50, 60, 70, 80]);
-    expect(data).toEqual(expected);
-  });
-
-  it('throws for scalars', done => {
-    // tslint:disable-next-line:no-any
-    expectPromiseToFail(() => tf.browser.toPixels(tf.scalar(1) as any), done);
-  });
-
-  it('throws for rank-1 tensors', done => {
-    expectPromiseToFail(
-        // tslint:disable-next-line:no-any
-        () => tf.browser.toPixels(tf.tensor1d([1]) as any), done);
-  });
-  it('throws for rank-4 tensors', done => {
-    expectPromiseToFail(
-        // tslint:disable-next-line:no-any
-        () => tf.browser.toPixels(tf.tensor4d([1], [1, 1, 1, 1]) as any), done);
-  });
-  it('throws for bool dtype', done => {
-    expectPromiseToFail(
-        () => tf.browser.toPixels(tf.tensor2d([1], [1, 1], 'bool')), done);
-  });
-  it('throws for rank-3 depth = 2', done => {
-    expectPromiseToFail(
-        () => tf.browser.toPixels(tf.tensor3d([1, 2], [1, 1, 2])), done);
-  });
-  it('throws for rank-3 depth = 5', done => {
-    expectPromiseToFail(
-        () => tf.browser.toPixels(tf.tensor3d([1, 2, 3, 4, 5], [1, 1, 5])),
-        done);
-  });
-  it('throws for float32 tensor with values not in [0 - 1]', done => {
-    expectPromiseToFail(
-        () => tf.browser.toPixels(tf.tensor2d([-1, .5], [1, 2])), done);
-  });
-  it('throws for int32 tensor with values not in [0 - 255]', done => {
-    expectPromiseToFail(
-        () => tf.browser.toPixels(tf.tensor2d([-1, 100], [1, 2], 'int32')),
-        done);
-  });
-  it('throws when passed a non-tensor', done => {
-    // tslint:disable-next-line:no-any
-    expectPromiseToFail(() => tf.browser.toPixels({} as any), done);
-  });
-
-  it('accepts a tensor-like object', async () => {
-    const x = [[10], [20]];  // 2x1;
-    const data = await tf.browser.toPixels(x);
+describeWithFlags('toPixels no canvas, returns void', ALL_ENVS, () => {
+  toPixelsTestCase(toPixelsNoCanvas, false);
+});
```

Comment:
`toPixelsNoCanvas` always returns data. I don't think we need this test case.

Reply:
Done, thanks